includes/shortcodes/class-sensei-shortcode-user-courses.php

```diff
@@ -293,7 +301,10 @@ public function render() {
 		ob_start();
 		echo '<section id="sensei-user-courses">';
 
-		Sensei_Messages::the_my_messages_link();
+		if ( empty( $this->use_blocks ) ) {
```

Comment:
Is there something not committed yet for this? I didn't find where this variable is set.